documents/retail-operations/workflow/job-management/job-queueing.md

```diff
@@ -134,12 +138,10 @@ Clicking `View Details` redirects retailers to a detailed logs page where they c
 
 - Access individual logs with specifics like start and finished date and time, user information, and unique log IDs.
 - Review the current status of each file (Finished, Running, Failed, Pending).
-- Download failed records as separate assets for further analysis.
+- Download original file and failed records for further analysis.
```

Comment:
```suggestion
- Download the original file and failed records for further analysis.
```

Reply:
this was written correctly when you copied it and you created an error when you pasted it.